lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1112,31 +1125,14 @@ sub save_user {
             );
            $request->{pls_import} = 1;
 
-           @{$request->{salutations}}
-            = $request->call_procedure(funcname => 'person__list_salutations' );
-
-           @{$request->{countries}}
-              = $request->call_procedure(funcname => 'location_list_country' );
+           $duplicate = _render_user($request);
 
-           my $locale = $request->{_locale};
-
-           @{$request->{perm_sets}} = (
-               {id => '0', label => $locale->text('Manage Users')},
-               {id => '1', label => $locale->text('Full Permissions')},
-           );
-           my $template = LedgerSMB::Template->new_UI(
-               $request,
-               template => 'setup/new_user',
-           );
-           $duplicate = $template->render_to_psgi($request);
+           return $duplicate
+               if $duplicate;
```

Comment:
Duplicate will never be false, right?

Reply:
Not unless something very wrong happens in the rendering.